src/unit_tests/data_export.rs

```diff
@@ -0,0 +1,134 @@
+use crate::models::common::Loadable;
+use crate::models::ctx::Ctx;
+use crate::models::data_export::DataExport;
+use crate::runtime::msg::{Action, ActionLoad};
+use crate::runtime::{Env, EnvFutureExt, Runtime, RuntimeAction, RuntimeEvent, TryEnvFuture};
+use crate::types::api::{APIResult, DataExportResponse};
+use crate::types::profile::{Auth, AuthKey, GDPRConsent, User};
+use crate::unit_tests::{
+    default_fetch_handler, Request, TestEnv, EVENTS, FETCH_HANDLER, REQUESTS, STATES,
+};
+use assert_matches::assert_matches;
+use enclose::enclose;
+use futures::future;
+use std::any::Any;
+use std::sync::{Arc, RwLock};
+use stremio_derive::Model;
+
+#[derive(Model, Default, Debug, Clone)]
+#[model(TestEnv)]
+struct TestModel {
+    ctx: Ctx,
+    data_export: DataExport,
+}
+
+fn data_export_fetch_handler(request: Request) -> TryEnvFuture<Box<dyn Any + Send>> {
+    match request {
+        Request {
+            url, method, body, ..
+        } if url == "https://api.strem.io/api/dataExport"
+            && method == "POST"
+            && &body == r#"{"type":"DataExport","authKey":"user_key"}"# =>
+        {
+            future::ok(Box::new(APIResult::Ok {
+                result: DataExportResponse {
+                    export_id: "user_export_id".into(),
+                },
+            }) as Box<dyn Any + Send>)
+            .boxed_env()
+        }
+        _ => default_fetch_handler(request),
+    }
+}
+
+#[test]
+fn data_export_with_user() {
+    let _env_mutex = TestEnv::reset();
+    *FETCH_HANDLER.write().unwrap() = Box::new(data_export_fetch_handler);
+    let mut ctx = Ctx::default();
+    ctx.profile.auth = Some(Auth {
+        key: AuthKey("user_key".into()),
+        user: User {
```

Comment:
you can use User::default() for the tests

Reply:
Fixed.